backend/app2/api/routes/soundfonts.py

```
@@ -4,89 +4,44 @@
-        "updated_at": datetime.utcnow()
-    }
-]
-
-@router.get("", response_model=List[Soundfont])
-@router.get("/", response_model=List[Soundfont])
+@router.get("", response_model=List[InstrumentFile])
+@router.get("/public", response_model=List[InstrumentFile])
 async def get_soundfonts(
-    current_user: Dict[str, Any] = Depends(get_current_user)
-) -> Any:
+    current_user: Dict[str, Any] = Depends(get_current_user),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)